projects/Allwinner/linux/linux.arm.conf

```diff
@@ -326,12 +326,12 @@ CONFIG_ARCH_MULTI_V6_V7=y
 # CONFIG_ARCH_STI is not set
 # CONFIG_ARCH_STM32 is not set
 CONFIG_ARCH_SUNXI=y
-# CONFIG_MACH_SUN4I is not set
-# CONFIG_MACH_SUN5I is not set
-# CONFIG_MACH_SUN6I is not set
-# CONFIG_MACH_SUN7I is not set
+CONFIG_MACH_SUN4I=y
+CONFIG_MACH_SUN5I=y
+CONFIG_MACH_SUN6I=y
```

Comment:
Is it done on purpose to enable other arch than A20?

Reply:
Yes because in theory all those family could just work with adding appropriate U-Boot config/DT file name combo. However, I have only A20 boards from those older platforms. If this is seen as unneeded, I can remove those entries.

Reply:
I'll remove them because I can't be sure they work as intended.